lib/wash_out/wsse.rb

```diff
@@ -62,11 +61,11 @@ def matches_expected_digest?(password)
       flavors = Array.new
 
       # Ruby / Savon
-      token = nonce + timestamp.to_s + expected_password
+      token = nonce + timestamp.strftime("%Y-%m-%dT%H:%M:%SZ") + expected_password
```

Comment:
Here I've assumed that a WSSE WSU createdAt will always be an xs:dateTime expressed in the canconcial way ( http://www.w3.org/TR/xmlschema-2/#dateTime - "The mapping so defined is one-to-one, except that '+00:00', '-00:00', and 'Z' all represent the same zero-length duration timezone, UTC; 'Z' is its canonical representation.")